apollo-server/v4/generated-types/tsconfig.json

```
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "rootDirs": ["src"],
+    "baseUrl": "./src",
+    "outDir": "dist",
+    "lib": ["es2020", "DOM"],
```

We shouldn't need DOM types for this example. If this raises any errors it might indicate a different problem, let me know if it's problematic!

```suggestion
    "lib": ["es2020"],
```